e2e/testcafe-devextreme/tests/accessibility/menu.ts

```diff
@@ -42,17 +42,24 @@ const options: Options<Properties> = {
   items: [items],
   disabled: [true, false],
   orientation: ['horizontal', 'vertical'],
+  adaptivityEnabled: [true, false],
 };
 
 const created = async (t: TestController, optionConfiguration): Promise<void> => {
-  const { disabled } = optionConfiguration;
+  const { disabled, adaptivityEnabled } = optionConfiguration;
 
   if (disabled) {
     return;
   }
 
   const menu = new Menu();
 
+  if (adaptivityEnabled) {
+    await t.resizeWindow(400, 800);
```

Comment:
@EugeniyKiyashko  is it okay to resize window directly, shouldn't some safe util be used here?

Reply:
I would prefer to change the menu width to emulate adaptivityEnabled mode rather than resizing the window

Reply:
Discussed privately - let's better set menu's `width` option to toggle mode that resize window

Reply:
[c317a55](https://github.com/DevExpress/DevExtreme/pull/28066/commits/c317a5579c7808781166a9bf40be333ba05ac031)